index.html

```diff
@@ -1,544 +1,627 @@
-                </a>
-              </span>
-              <span class="navbar-item">
-                <a class="button is-white is-outlined">
-                  <span class="icon">
-                    <i class="fab fa-github"></i>
-                  </span>
-                  <span>View Source</span>
-                </a>
-              </span>
-            </ul>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Web Accessibility 2022</title>
+    <link rel='stylesheet prefetch'
+      href='https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.2/css/bulma.min.css'>
+    <link rel='stylesheet prefetch'
+      href='https://use.fontawesome.com/releases/v5.0.10/css/all.css'>
+    <link rel="stylesheet prefetch" href="css/styles.css">
+  </head>
 
-  <div class="hero-bg hero-body is-fullheight">
-    <div class="container has-text-centered">
-      <div class="column is-10 is-offset-1">
-        <div class="title has-text-white">We have web accessibility in mind</div>
-      </div>
-    </div>
-  </div>
-
-  <div class="section" id="subscription">
-    <div class="section-heading has-text-centered">
-      <div class="column is-8 is-offset-2">
-      <div class="title is-2">Would you like to get the latest news?</div>
-        <div class="subtitle is-5">Subscribe to our events, updates and more</div>
-      <div class="box has-background-grey-light">
-        <div class="field is-grouped">
-          <p class="control is-expanded">
-            <input class="input">
-          </p>
-          <p class="control">
-            <a class="button is-info">Notify Me</a>
-          </p>
-        </div>
-      </div>
-      </div>
-    </div>
-
-  <div class="section" id="about">
-    <div class="section-heading has-text-centered">
-      <div class="title is-2">About Us</div>
-      <div class="subtitle is-5">We of all trades, master of "some"</div>
-      <div class="container">
-        <p>Web developers with more than <b>4 years</b> of well-rounded experience with a degree in the
-          field of
-          <b>Computer Science</b>, extensive knowledge of modern Web techniques and love for <b>Coffee</b>.
-          Looking for an opportunity to work and upgrade, as well as being involved in an organization that
-          believes
-          in gaining a competitive edge and giving back to the community.</p>
-      </div>
-    </div>
-
-    <div class="content container">
-      <div class="is-1 title">OUR HISTORY</div>
-      <p>Since 1989<sup><a>[1]</a></sup>, we deliver high quality solutions to make your business more accessible.</p>
-      <div class="is-2 title">1989</div>
-      <p>On February 1989, seventeen people met to talk, ski, relax, and try to find common ground.</p>
+  <body>
+    <nav aria-label="Skip navigation links" class="visually-hidden">
```

Comment:
If you add this class to skip link, then it won't be available to the users and it loses its purpose
<img width="1227" alt="image" src="https://github.com/yurassic/accessibility-training/assets/28926420/a6a8edf3-f88d-4b3c-9e42-8e9b6571f0e2">
If it is available, the user will get a change to get directly to the main content

Reply:
The purpose of this navigation is only for users who use screenreaders, so they will have ability to navigate to main menu directly, so it's hidden for most users.

Reply:
No worries